templates/crds/addressplans.crd.yaml

```diff
@@ -90,6 +90,16 @@ spec:
           type: string
         addressType:
           type: string
+        ttl:
+          type: object
+          description: "Restricts message time-to-live (TTL). Applies to address types queue and topic only."
```

Comment:
The description should include the expected unit for maximum and minimum values.

Reply:
Done.